oio/account/backend.py

```diff
@@ -314,6 +315,26 @@ class AccountBackend(RedisConnection):
         end;
         """
 
+    lua_get_extended_container_info = """
+        local ckey = KEYS[1]
+        local bucket_prefix = ARGV[1]
+        local bname = redis.call('HGET', ckey, 'bucket')
+
+        local replication_enabled = 'false'
```

Comment:
Instead of adding an option (`check_replication_enabled`), why not set the default value to true?

Reply:
If we enable it by default, any unconfigured bucket will be replicated. The user would have to explicitly disable it for each bucket. That's not what we want.

And also, by disabling the check, legacy platforms won't see any performance degradation.